cve_bin_tool/checkers/ppp.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for ppp
+
+https://www.cvedetails.com/vulnerability-list/vendor_id-102/product_id-2091/Samba-PPP.html
+
+"""
+import os
+from ..util import regex_find
+
+
+def get_version(lines, filename):
+    """returns version information for ppp as found in a given file.
+    The version info is returned as a tuple:
+        [modulename, is_or_contains, version]
+
+    modulename will be ppp if ppp is found (and blank otherwise)
+    is_or_contains indicates if the file is a copy of ppp or contains one
+    version gives the actual version number
+
+    VPkg: samba, ppp
+    """
+    regex = [r"pppd\/([0-9]+\.[0-9]+\.[0-9]+)"]
```

Comment:
As I mentioned previously, strings that look ilke directories make me nervous because past experience tells me that they're very distribution specific.  I will not be merging this until there are tests showing that it works with multiple distritubions (and this may require more regex-signatures before it will work).

Reply:
I will try to find some better strings to be added to regex.